airflow/README.md

```diff
@@ -83,3 +83,5 @@ Note that the following variables were set manually in cloud composer:
 * `SENDGRID_MAIL_FROM`
 * `POD_CLUSTER_NAME` - name of the kubernetes cluster
 * `POD_LOCATION` - location of cluster (e.g. us-west-2a)
+* `CALITP_SLACK_URL` - slack webhook api url, for posting errors to a channel.
```

Comment:
what channel does this point at?

Reply:
there's an environment variable in the composer prod environment with a slack webhook URL that points to the #data-infra-notify channel.